app/templates/components/forms/admin/content/pages-form.hbs

```diff
@@ -46,7 +45,7 @@
     {{/ui-dropdown}}
   </div>
   <div class="ui buttons">
-    <button class="ui teal button" type="submit">{{t 'Update Page'}}</button>
-    <button class="ui red button">{{t 'Delete Page'}}</button>
+    <button class="ui teal button" type="submit">{{saveButton}}</button>
```

Comment:
You can do that in the template itself. Don't pass in variables for all these sort of things please

have a simple if check 

`{{if something (t 'Create page') (t 'Update page')}}`